tools/devops/automation/build-pipeline.yml

```diff
@@ -241,6 +243,63 @@ stages:
         xqaCertPass: $(xqa--certificates--password)
         enableDotnet: ${{ parameters.enableDotnet }}
 
+# .NET 6 Release Prep and VS Insertion Stages
+- stage: prepare_release
+  displayName: Prepare Release
+  dependsOn: build_packages
+  condition: and(or(eq(dependencies.build_packages.result, 'Succeeded'), eq(dependencies.build_packages.result, 'SucceededWithIssues')), eq(variables.IsPRBuild, 'False'), eq(${{ parameters.enableDotnet }}, true))
+  jobs:
+  # Check - "xamarin-macios (Prepare Release Sign NuGets)"
+  - template: sign-artifacts/jobs/v2.yml@templates
+    parameters:
+      artifactName: package
+      signType: Real
+      usePipelineArtifactTasks: true
+
+  # Check - "xamarin-macios (Prepare Release Convert NuGet to MSI)"
+  - template: nuget-msi-convert/job/v1.yml@templates
+    parameters:
+      yamlResourceName: templates
+      dependsOn: signing
+      artifactName: nuget-signed
+      artifactPatterns: | # Include MacCatalyst, but not macOS packs?
+        !*macOS*
```

Comment:
@rolfbjarne @Redth @jonathanpeppers Do you have thoughts on which maciOS workloads should we include in VS Windows? Right now this tooling is picking up iOS, tvOS, and Mac Catalyst. I was thinking we should at least insert everything that MAUI depends on, perhaps we should only insert iOS and Mac Catalyst on Windows?

Reply:
I would think it would probably be fine to not insert the macOS packs in VS? What about tvOS?

A dev could still *get* the packs from `dotnet workload install` if needed, but I don't think we need a checkbox & component to show up in the VS installer.

So far, I've only heard about "remote build" support for iOS & MacCatalyst, and MAUI only uses these two workloads so far:

* https://github.com/dotnet/maui/blob/c3083b9d9d229ff221483a03003eb5bd53692503/src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.json#L53-L56
* https://github.com/dotnet/maui/blob/c3083b9d9d229ff221483a03003eb5bd53692503/src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.json#L68-L71

Reply:
I'm pretty sure we don't have any support for `macOS` in VS Windows (so it wouldn't make sense to add them to the installer).

I'm not sure we have tvOS support either, but I've never heard anything either way.

@emaf can confirm though